guides/voice/gather-dtmf-tones-guide/gather-example-step-0/example.5.x.cs

```diff
@@ -3,20 +3,22 @@
 
 using System.Web.Mvc;
 using Twilio.AspNet.Mvc;
+using Twilio.AspNet.Common;
 using Twilio.TwiML;
+using System;
 
 public class VoiceController : TwilioController
 {
     [HttpPost]
-    public ActionResult Index()
+    public TwiMLResult Index()
```

Comment:
Is TwiMLResult the "new" ActionResult? (I don't know much about Voice apps.)

Reply:
Yes, this uses the [asp-net helper library](https://github.com/twilio/twilio-aspnet) which includes TwiMLResult. TwiMLResult is derived from ActionResult for the [Twilio.AspNet.Mvc library](https://github.com/twilio/twilio-aspnet/blob/9bc93ce0db478a67a2b0071f9691d18b9d40ffa9/src/Twilio.AspNet.Mvc/TwiMLResult.cs#L10) and [IActionResult for the Twilio.AspNet.Core library](https://github.com/twilio/twilio-aspnet/blob/9bc93ce0db478a67a2b0071f9691d18b9d40ffa9/src/Twilio.AspNet.Core/TwiMLResult.cs#L8)